app/controllers/auth0_controller.rb

```diff
@@ -48,11 +48,11 @@ def failure
   # Handle omniauth errors coming from Auth0.
   def omniauth_failure
     # Error and error_description come from Auth0. Ex: unauthorized and password_expired.
-    unless params["error"] && params["error_description"]
-      LogUtil.log_err_and_airbrake("omniauth_failure called with missing error or error_description")
-    end
     error_type = (params["error"] || "").to_sym
     error_code = (params["error_description"] || "").to_sym
+    unless params["error"] && params["error_description"]
+      LogUtil.log_err_and_airbrake("omniauth_failure called with missing error or error_description. error=#{error_type}, error_description=#{error_description}")
```

Comment:
did you mean `error_description=#{error_code}` ?

Reply:
Oh, thank you for catching this. I couldn't test this part of the code (not sure how to reproduce this error)